CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-## 1.2.0 (not yet released)
+## 1.2.1 (not yet released)
```

Comment:
the next version is going to be 1.1.1 so these changes can be moved under 1.1.1 (and the 1.2.1 target can be removed)